.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+on:
+  push:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+      GOPATH: /home/runner/work/botkube
+      GOBIN: /home/runner/work/botkube/bin
```

Comment:
What is $HOME value? Can we use $HOME to set the path?

Reply:
$HOME value is /home/runner. We can use that for $GOPATH.

Reply:
the value for $HOME is /home/runner but it gives error if we try to assign value of GOPATH to a relative path. 
Check [error](https://github.com/ameydev/botkube/runs/435702976?check_suite_focus=true#step:4:11).
so I think we need to give absolute path which would  be `/home/runner` or can keep the same value.